### PR TITLE
feat(FileInput): add onFocus and onBlur events to FileInput

### PR DESCRIPTION
**Pull request by rhuankarlus**
[FX-2567](https://toptal-core.atlassian.net/browse/FX-2567)

### Description

In this PR the `onFocus` and `onBlur` events are being added to the [FileInput](https://picasso.toptal.net/?path=/story/forms-fileinput--fileinput) component. The goal is to validate this field when the user tabs over it creating a better UX.

### How to test

- CI green
- Go to the [Form Default Story](https://picasso.toptal.net/?path=/story/picasso-forms-form--form#default)
- Press tab to navigate over the components
- Note that, when tab is pressed on the **FileInput** component, the requireness message appears

### Video

https://user-images.githubusercontent.com/4607181/160970449-b7daa106-dcd4-49dc-a319-c7ab36abc2b2.mp4

### Review

- [ ] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [ ] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [ ] Ensure that deployed demo has expected results and good examples
- [ ] Ensure that tests pass by running `yarn test`
- [ ] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [ ] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

**Review by augustobmoura — APPROVED** [review]
LGTM! :+1:

**Comment by augustobmoura** [comment]
Just remember to add the [changeset](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md#release)

File: packages/picasso/src/FileInput/FileInput.tsx [hunk 1]
```diff
@@ -62,6 +70,8 @@ export const FileInput = forwardRef<HTMLInputElement, Props>(function FileInput(
         variant='secondary'
         disabled={Boolean(disabled || preventAddingNewFiles)}
         onClick={() => inputRef.current && inputRef.current.click()}
+        onFocus={onFocus}
```
Comment (denieler) [inline packages/picasso/src/FileInput/FileInput.tsx:73]: shouldn't we add them to the wrapper `Container` root component? I think in that case implementation will be more flexible (for example in a case we would change the design and more elements will appear in this component)

**Review by TomasSlama — APPROVED** [review]

**Review by OndrejTuma — APPROVED** [review]
Tested locally. Works great 👍

**Review by denieler — APPROVED** [review]

**Pull request merged**